tests/tools.vitruv.applications.pcmumlclass.tests/src/tools/vitruv/applications/pcmumlclass/tests/MediaStoreRepositoryCreationTest.xtend

```diff
@@ -53,38 +73,29 @@ class MediaStoreRepositoryCreationTest extends LegacyPcmUmlClassApplicationTest
 	}
 
 	@Test
-	@Disabled // skip until needed because of performance
 	def void testMediaStoreRepositoryCreation_UML2PCM() {
-		val uri = URI.createURI(UML_MEDIA_STORE_REPOSITORY_PATH)
-		val umlMsRepositoryResource = uri.testResource
-		assertNotNull(umlMsRepositoryResource)
-		val umlMsModel = umlMsRepositoryResource.contents.head as Model
-		assertNotNull(umlMsModel)
+		var umlMsModel = removePrimitiveTypes(loadUmlMsModel)
```

Comment:
Is this related to the UML primitive type issues we currently face? If so, please add an explaining comment.